src/Elasticsearch/State/CollectionProvider.php

```diff
@@ -26,7 +26,7 @@
 /**
  * Collection provider for Elasticsearch.
  *
- * @experimental
+ * @deprecated
```

Comment:
same as for the ItemProvider I think that this code was fine it just needs to use the new option.

Reply:
https://github.com/api-platform/core/pull/5297#discussion_r1060353458